packages/n/noto-sans-ttf/package.yml

```diff
@@ -41,5 +41,9 @@ install    : |
 
     install -dm00755 $installdir/usr/share/fonts/conf.d
     ln -sv /usr/share/fontconfig/conf.avail/66-noto-color-emoji.conf $installdir/usr/share/fonts/conf.d/66-noto-color-emoji.conf
+
+    install -Dm00644 $pkgfiles/noto-sans-ttf.metainfo.xml $installdir/usr/share/metainfo/noto-sans-ttf.metainfo.xml
```

Comment:
Needs to be patterned into the `noto-sans-ttf`

Reply:
I'm not sure what "patterned into the package" means or the steps needed to accomplish this. I read over [the pattern docs](https://help.getsol.us/docs/packaging/packaging-practices#patterns) but I don't know what to put in the patterns section. Might I ask you to elaborate?

Reply:
Confirmed with Staudey and testing this should not be added.